src/rpc/handlers/Tx.cpp

```diff
@@ -36,6 +36,18 @@ doTx(Context const& context)
     if (!range)
         return Status{RippledError::rpcNOT_READY};
 
+    if (request.contains(JS(min_ledger)) && request.contains(JS(max_ledger)) &&
+        request.at(JS(min_ledger)).is_int64() &&
+        request.at(JS(max_ledger)).is_int64())
+    {
+        const auto& minLedger = request.at(JS(min_ledger)).as_int64();
+        const auto& maxLedger = request.at(JS(max_ledger)).as_int64();
+
+        if (maxLedger - minLedger > 1000 && maxLedger > minLedger)
```

Comment:
Looks like I'm allowed to pass negative ledgers.

Reply:
And most importantly - this code still does a check for nothing - we don't use the parameters we check. This means you have to go to rippled codebase and see how this stuff works exactly and if we can/should port it to clio.

Reply:
@godexsoft for context as outlined in the original issue [here](https://github.com/XRPLF/clio/issues/278), there was no previous check in Rippled to determine if there was a difference of 1000 or more between the optional min_ledger and max_ledger parameters in the doc [here](https://xrpl.org/tx.html). 

According to the documentation, specifying min_ledger and max_ledger would return the set of transactions between those two. In rippled's Tx.cpp, the min_ledger and max_ledger would be formed into pairs and then passed as a separate call. 

Should we do the same for Clio, or pass in the tx set differently?
![image](https://user-images.githubusercontent.com/47958858/202072689-ea6bec7a-dbc5-44b5-9147-a82ea85a15d4.png)

Reply:
@godexsoft this issue is getting taken over by CJ due to shared concerns about missing min_ledger and max_ledger handling functionality in Clio that exists in Rippled